six.py

```diff
@@ -102,9 +102,9 @@ def __get__(self, obj, tp):
 
 class MovedModule(_LazyDescr):
 
-    def __init__(self, name, old, new=None):
+    def __init__(self, name, old, new=None, use_new=True):
```

Comment:
How does this help?

Reply:
Sorry, I see now. Perhaps you should pass use_new in by name to make the code below clear.

Reply:
Would it be better to parametrise by "version_changed" and do the comparison here?
Or leave it all on the call side, but use
```
MovedModule("collections_abc", "collections", "collections.abc" if sys.version_info[0:2] >= (3, 3) else "collections")
```

Reply:
Let's wait to see what @benjaminp says before we change anything, I only use that parameter because any Python 3 version below 3.3 doesn't include the "abc" submodule.

Reply:
Alternatively to a boolean `use_new` one could add an argument that specifies from which version on, the new behavior should be used, e.g. `since='3.3'` or `new_since='3.3'`.